services/QuillLMS/app/models/student_problem_report.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: student_problem_reports
+#
+#  id                  :bigint           not null, primary key
+#  report              :string           not null
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  feedback_history_id :bigint
+#
+# Indexes
+#
+#  index_student_problem_reports_on_feedback_history_id  (feedback_history_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (feedback_history_id => feedback_histories.id)
+#
+class StudentProblemReport < ApplicationRecord
+  belongs_to :feedback_history
+
+  validates_presence_of :feedback_history_id
```

Comment:
Super minor, but since we're making `report` a `NOT NULL` field in the migration, it might be worth adding a `validates_presences_of :report` here as well.